src/blueTheme.ts

```diff
@@ -6,88 +6,88 @@ All rights reserved.
+        secondary: BLUIColors.primary[40],
+        onSecondary: BLUIColors.primary[100],
+        secondaryContainer: BLUIColors.primary[80],
+        onSecondaryContainer: BLUIColors.primary[30],
+        tertiary: BLUIColors.primary[40],
+        onTertiary: BLUIColors.primary[100],
+        tertiaryContainer: BLUIColors.primary[80],
+        onTertiaryContainer: BLUIColors.primary[30],
+        error: BLUIColors.error[40],
+        errorNonText: BLUIColors.error[50],
+        onError: BLUIColors.primary[100],
+        errorContainer: BLUIColors.error[90],
+        onErrorContainer: BLUIColors.error[30],
+        background: BLUIColors.neutral[98],
+        onBackground: BLUIColors.neutral[10],
+        surface: BLUIColors.neutral[98],
+        onSurface: BLUIColors.neutral[10],
+        surfaceVariant: BLUIColors.neutralVariant[90],
+        onSurfaceVariant: BLUIColors.neutralVariant[30],
+        outline: BLUIColors.neutralVariant[50],
+        outlineVariant: Color(BLUIColors.neutralVariant[50]).alpha(0.25),
+        shadow: BLUIColors.primary[0],
+        scrim: BLUIColors.primary[0],
+        inverseSurface: 'rgb(47, 48, 51)',
+        inverseOnSurface: 'rgb(241, 240, 244)',
+        inversePrimary: 'rgb(169, 199, 255)',
+        elevation: {
+            level0: 'transparent',
+            level1: BLUIColors.primary[100],
+            level2: BLUIColors.neutral[97],
+            level3: BLUIColors.neutral[94],
+            level4: BLUIColors.neutral[92],
+            level5: BLUIColors.neutral[90],
         },
+        surfaceDisabled: 'rgba(26, 27, 30, 0.12)',
+        onSurfaceDisabled: 'rgba(26, 27, 30, 0.38)',
+        backdrop: 'rgba(45, 48, 56, 0.4)',
+        disabled: BLUIColors.neutral[10],
+        surfaceContainerLow: BLUIColors.neutral[97],
+        neutral10: BLUIColors.neutralVariant[10],
+        neutral30: BLUIColors.neutral[30],
+        neutral80: BLUIColors.neutral[80],
+        neutral100: BLUIColors.primary[100],
+        warning: BLUIColors.warning[40],
+        warningNonText: BLUIColors.warning[60],
+        onWarning: BLUIColors.primary[100],
+        warningContainer: BLUIColors.warning[90],
+        onWarningContainer: BLUIColors.warning[30],
+        success: BLUIColors.success[40],
+        successNonText: BLUIColors.success[50],
+        onSuccess: BLUIColors.primary[100],
+        successContainer: BLUIColors.success[90],
+        onSuccessContainer: BLUIColors.success[30],
+        orange: BLUIColors.orange[40],
+        orangeNonText: BLUIColors.orange[50],
+        onOrange: BLUIColors.primary[100],
+        orangeContainer: BLUIColors.orange[90],
+        onOrangeContainer: BLUIColors.orange[30],
+        purple: BLUIColors.purple[40],
+        purpleNonText: BLUIColors.purple[50],
+        onPurple: BLUIColors.primary[100],
+        purpleContainer: BLUIColors.purple[90],
+        onPurpleContainer: BLUIColors.purple[30],
+        primaryNonText: BLUIColors.primary[50],
```

Comment:
group these with all other "primary" colors on the top to make it easier to maintain in the future